examples/command-model/src/main/java/gov/nasa/jpl/aerie/command_model/sequencing/command_dictionary/CompilingDictionaryImpl.java

```diff
@@ -0,0 +1,40 @@
+package gov.nasa.jpl.aerie.command_model.sequencing.command_dictionary;
+
+import gov.nasa.jpl.aerie.command_model.activities.commands.*;
+import gov.nasa.jpl.aerie.command_model.sequencing.Command;
+
+public class CompilingDictionaryImpl implements CompilingDictionary {
+
+    @Override
+    public CommandActivity interpret(Command command) {
+        // A real definition of this might want to use reflection + a read-in command dictionary file
```

Comment:
Not sure we need this file at all. I would be a proponent of automatically generating these returns via the value mappers: 
```
    @Override
    public CommandActivity interpret(Command command) {
      var valueMapper = ActivityTypes.directiveTypes.get(command.stem());
      var outputType = valueMapper.getInputAsOutput().getSchema();
      var argumentMap = IntStream.range(0, valueMapper.getInputType().getParameters().size()).boxed().collect(
          Collectors.toMap(i -> valueMapper.getInputType().getParameters().get(i).name(), 
                           i -> SerializedValue.of(command.arguments().get(i)))); // might be able to use the parameter value schema?
      try{
        return (CommandActivity) valueMapper.getInputType().instantiate((Map<String, SerializedValue>) argumentMap);
      } catch(Exception e) {
        throw new RuntimeException("Stem %s is not a recognized command.".formatted(command.stem()));
      }
    }
```

Reply:
I really like your approach - get the value mappers to do 90% of the heavy lifting.
You might need a layer of translation between the format coming in, which should be the format the mission is authoring sequences in, and the JSON format the value mapper expects, but we could figure that out as we go. I expect that would be a fairly simple transformation.

A couple things to think about there, which I completely gloss over in this toy example:
1. Timing codes - I assume each command starts when the last one ends, but really we need to interpret the timing information. Maybe that's handled by an abstract base class for commands, which adds the time tag as a field and the timing behavior as a sort of "pre-behavior" of the command?
2. Variables - If we're hewing the seq JSON style, variables-are-arguments syntax like `SEQ_ECHO L00STR`, we need a way to represent that. That suggests that maybe command arguments are a type unto themselves, not just primitives? Alternatively, we use the `SEQ_VAR_CMD` expansion, so we represent the same example something like `SEQ_VAR_CMD <opcode for SEQ_ECHO> L00STR` and directly replicate what FSW does - the SEQ_VAR_CMD command looks up the opcode and variable value, then calls that command.

Reply:
> That suggests that maybe command arguments are a type unto themselves, not just primitives?

I think this is generally a good practice. At the least, it's helpful to differentiate in FCPL between literals and symbols, and I expect a more general pattern that could apply to other sequencing languages/FSWs would also be improved by breaking the requirement that all allowable argument types be mapped to Java primitives.